.github/workflows/scripts/handle_deleted_files.sh

```diff
@@ -0,0 +1,37 @@
+"""
+Functionality for handling deleted files from the repository. Once a file is deleted, the csvcubed build and inspect command outputs generated for that file will also be deleted.
+"""
+
+echo "::set-output name=has_outputs::false"
+
+echo "Input argument: $1"
+IFS=', ' read -r -a deleted_files <<<"$1"
+echo "deleted_files: ${deleted_files[@]}"
+
+for file in "${deleted_files[@]}"; do
+  file_path="${file%.*}"
+  file_name="${file_path##*/}"
+  file_extension="${file##*.}"
+
+  if [[ $file_extension == "csv" && "${file_path}" != *"out/"* ]]; then
+    echo $'\n'
+    echo "Handling deletions for file ${file}"
+    config_file="${file_path}.${file_extension}.json"
+    out_folder="out/${file_path}/"
+
+    if [[ -f $config_file ]]; then
+      git rm $config_file
+      git commit -m "Deleted out folder for file ${file} - $(date +'%d-%m-%Y at %H:%M:%S')"
```

Comment:
I would be tempted to only have one commit for both of these changes. At any rate, this commit message isn't right since you're deleting the config file. Also, I would be hesitant to delete the qube-config.json file if the user hasn't done it themselves. It's not an output that we've automatically generated.